src/write.c

```diff
@@ -2900,7 +2909,7 @@ avifResult avifEncoderFinish(avifEncoder * encoder, avifRWData * output)
 
     avifEncoderItemReferenceArray layeredColorItems;
     avifEncoderItemReferenceArray layeredAlphaItems;
-    avifResult result = AVIF_RESULT_OK;
+    result = AVIF_RESULT_OK;
```

Comment:
Note that `result` is already `AVIF_RESULT_OK` here, but it is fine to enforce it to be clear.

Reply:
If it is unassigned, it will be caught by the compiler anyway. Removing it.